### ISSUE TITLE
training dataset and super resolution capability  

### ISSUE THREAD

**Issue by jimmycv07** [KAIST-VICLab/FMA-Net#10]
Hi, thank you for your great work! I have a couple of questions:

1. Are the experimental results' pretrained weights trained on the REDS train set (excluding Clips 000, 011, 015, and 020) or on the REDS train, validation, and test sets (excluding Clips 000, 011, 015, and 020)?

2. Can FMA-Net handle custom time super resolution, such as factors other than 4x?

Thank you in advance for your response.

**Comment by GeunhyukYouk** [comment]
Hi, thanks for your interest in FMA-Net!

1. The provided pretrained weights are trained on the REDS train (excluding 000, 011, 015, 020) and validation sets following previous works. The test set is excluded since ground truth is not provided. Please refer to this [code](https://github.com/KAIST-VICLab/FMA-Net/blob/main/preprocessing/generate_reds4.py).

2. FMA-Net can handle scale factors other than 4x. However, we don't provide pretrained models for this, so users need to train the model themselves. Specify the desired [scale factor](https://github.com/KAIST-VICLab/FMA-Net/blob/e07ddc5fd393bbe7f710c564420b1a8ef2f4e99f/experiment.cfg#L19), modify the [Pixel_Shuffle_Block](https://github.com/KAIST-VICLab/FMA-Net/blob/e07ddc5fd393bbe7f710c564420b1a8ef2f4e99f/model.py#L193) accordingly (this is implemented only for 4x upsampling), and proceed with training.

**Comment by jimmycv07** [comment]
Hi, thank you for your prompt and detailed response! I understand now!
